pkg/types/gcp/validation/platform.go

```diff
@@ -1,6 +1,7 @@
 package validation
 
 import (
+	"github.com/openshift/installer/pkg/types"
```

Comment:
Can we move this to be in the final import group? Those installer imports should be grouped together. Imports look a little strange in this file.

Reply:
Absolutely, looks like goland automatic import resolution has got me again!